webapp/src/domain/tests/Subscriptions.tsx

```diff
@@ -8,11 +8,13 @@ import { info, teams, User } from "../user/api"
 import { alertAction } from "../../alerts"
 import { teamToName, useTester } from "../../auth"
 import { TabFunctionsRef } from "../../components/SavedTabs"
+import { Checkbox } from "@patternfly/react-core"
 import UserSearch from "../../components/UserSearch"
 
 type SubscriptionsProps = {
     testId: number
     testOwner?: string
+    mutemissingruns: string
```

Comment:
why is this a string and not a `boolean`?

Reply:
changed